widgets/SensitiveRecordFilter-splittertransform.json

```diff
@@ -115,6 +115,59 @@
           }
         }
       ]
+    },
+    {
+      "label": "Advanced configuration",
+      "properties": [
+        {
+          "widget-type": "toggle",
+          "name": "dlp-override-enabled",
+          "label": "Custom DLP endpoint",
+          "widget-attributes": {
+            "default": "false",
+            "on": {
+              "value": "true",
+              "label": "Yes"
+            },
+            "off": {
+              "value": "false",
+              "label": "No"
+            }
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Host",
+          "name": "dlp-host",
+          "widget-attributes": {
+            "default": "dlp.local"
```

Comment:
is this a valid hostname?

Reply:
Yes, that's the hostname we agreed to use by default for local DLP. It will have to be mapped by a proxy to backend service.